pkg/appregistry/mock_adapter.go

```diff
@@ -34,16 +34,16 @@ func (m *MockapprApiAdapter) EXPECT() *MockapprApiAdapterMockRecorder {
 }
 
 // ListPackages mocks base method
-func (m *MockapprApiAdapter) ListPackages() (models.Packages, error) {
-	ret := m.ctrl.Call(m, "ListPackages")
+func (m *MockapprApiAdapter) ListPackages(namespace string) (models.Packages, error) {
```

Comment:
I'd rather have all these moved to a new package called generated, if you want to have the generated packages.